tools/codegen/api/lazy.py

```diff
@@ -34,6 +34,10 @@ def process_ir_type(typ: Type) -> Union[BaseCType, VectorCType, OptionalCType, L
             return BaseCType(valueT)
         elif typ.name == BaseTy.Scalar:
             return BaseCType(scalarT)
+        elif typ.name == BaseTy.int:
+            return BaseCType(intT)
+        elif typ.name == BaseTy.bool:
```

Comment:
I added this in one of my local patch too. haha. I will rebase on your patch then.